be2-scala/.github/workflows/be2-scala.yml

```diff
@@ -0,0 +1,29 @@
+name: PoP CI be2-scala
+
+on:
+  push:
+    branches: [ work-be2-* ]
+  pull_request:
+    branches: [ be2-scala ]
```

Comment:
we should change this to master right?

Reply:
Yes indeed! Didn't change it yet ^^' But I think there will be more things that change. For now, I'll only add "master" in pull_requests :)